Modules/Client/Resources/views/subviews/create-client-details.blade.php

```diff
@@ -58,10 +59,20 @@ class="checkbox-custom mb-1.9 mb-1.67 mr-3"
                 @endforeach
             </select>
         </div>
-
+        <div class="form-group col-md-5">
+            <label for="country_id">Country
+                <span data-toggle="tooltip" data-placement="right" title="If this client came via a channel partner then link that client from here."><i class="fa fa-question-circle"></i>&nbsp;</span>
+            </label>
+            <select name="country_id" id="country_id" class="form-coutrol">
```

Comment:
Check the naming convention used in portal from [here](https://github.com/ColoredCow/portal/blob/master/docs/variable-naming-convention.md)
```suggestion
            <select name="country_id" id="countryId" class="form-control">
```